.github/workflows/windows_conditional_compilation.yml

```diff
@@ -98,72 +88,226 @@ jobs:
+      - name: Cmake install - OpenVINO
+        run: |
+          cmake -DCMAKE_INSTALL_PREFIX=${{ env.INSTALL_DIR }} -P ${{ env.BUILD_DIR }}/cmake_install.cmake
+
+      - name: Build C++ samples - OpenVINO build tree
+        run: |
+          cmake -G "${{ env.CMAKE_GENERATOR }}" -DOpenVINO_DIR=${{ env.BUILD_DIR }} -S ${{ env.INSTALL_DIR }}/samples/cpp -B ${{ env.BUILD_DIR }}/cpp_samples
+          cmake --build ${{ env.BUILD_DIR }}/cpp_samples --parallel --config ${{ env.CMAKE_BUILD_TYPE }} --target hello_query_device
+
+      - name: Build C samples - OpenVINO install tree
+        run: |
+          & ${{ env.INSTALL_DIR }}/samples/c/build_samples_msvc.bat -i ${{ env.INSTALL_DIR }} -b ${{ env.BUILD_DIR }}/c_samples
+
+      - name: Ctest - OpenVINO unit tests
         shell: cmd
-        run: dir ${{ env.OPENVINO_REPO }}\bin\ /s
+        continue-on-error: true
+        run: |
+          set path=%path%;${{ env.OPENVINO_REPO }}\temp\tbb\bin
+          ctest -C ${{ env.CMAKE_BUILD_TYPE }} --test-dir ${{ env.BUILD_DIR }} -V -L UNIT
 
-      - name: Code usage analysis
+      - name: Perform code tracing via ITT collector
         shell: cmd
-        working-directory: ${{ env.OPENVINO_REPO }}
+        continue-on-error: true
         run: |
           set path=%path%;${{ env.OPENVINO_REPO }}\temp\tbb\bin
-          call "${{ env.VCVARSPATH }}" && python thirdparty\itt_collector\runtool\sea_runtool.py --bindir ${{ env.OPENVINO_REPO }}\bin\intel64\${{ env.BUILD_TYPE }} -o ${{ env.BUILD_DIR }}\itt_stat ! ${{ env.OPENVINO_REPO }}\bin\intel64\${{ env.BUILD_TYPE }}\benchmark_app.exe -niter 1 -nireq 1 -m ${{ env.MODELS_PATH }}\models\test_model\test_model_fp32.xml -d CPU
+          
+          python3 ${{ env.OPENVINO_REPO }}\thirdparty\itt_collector\runtool\sea_runtool.py ^
+          --bindir ${{ env.OPENVINO_REPO }}\bin\intel64\${{ env.CMAKE_BUILD_TYPE }} ^
+          -o ${{ env.SELECTIVE_BUILD_STAT_DIR }}\itt_stat ! ${{ env.OPENVINO_REPO }}\bin\intel64\${{ env.CMAKE_BUILD_TYPE }}\benchmark_app.exe ^
+          -niter 1 ^
+          -nireq 1 ^
+          -m ${{ env.MODELS_PATH }}\models\test_model\test_model_fp32.xml ^
+          -d CPU
 
-      - name: List csv files
+      - name: List bin files
         shell: cmd
-        run: dir ${{ env.BUILD_DIR }}\*.csv /s /p
+        run: dir ${{ env.OPENVINO_REPO }}\bin\ /s
+
+      - name: Pack Artifacts
+        continue-on-error: true
+        run: |
+          $file=Get-ChildItem -Path "${{ env.SELECTIVE_BUILD_STAT_DIR }}"
+          $compress = @{
+            Path = $file
+            CompressionLevel = "Optimal"
+            DestinationPath = "${{ env.BUILD_DIR }}/openvino_selective_build_stat.zip"
+          }
+          Compress-Archive @compress
+
+          $compress = @{
+            Path = "${{ env.OPENVINO_REPO }}/bin/intel64/${{ env.CMAKE_BUILD_TYPE }}/ov_cpu_func_tests.exe", "${{ env.OPENVINO_REPO }}/src/tests/test_utils/functional_test_utils/layer_tests_summary"
```

Comment:
I suppose you also need to package TBB libraries. You can install them via:
```
cmake -DCOMPONENT=tbb -P cmake_install.cmake
```
and package to test package for `CPU_Functional_Tests` job.
In the job itself, just set path to TBB dir with DLLs

Reply:
or maybe simpler - set `-DTHREADING=SEQ` to avoid TBB usage